shell/platform/fuchsia/flutter/gfx_external_view_embedder.cc

```diff
@@ -328,15 +328,13 @@ void GfxExternalViewEmbedder::SubmitFrame(
             kScenicZElevationBetweenLayers * scenic_layer_index +
             embedded_views_height;
 
-        if (view_mutators.total_transform != view_params.transformMatrix()) {
-          FML_LOG(ERROR) << "Failed assertion: view_mutators.total_transform "
-                            "!= view_params.transformMatrix()";
-          FML_LOG(ERROR) << "view_mutators.total_transform:";
-          view_mutators.total_transform.dump();
-          FML_LOG(ERROR) << "view_params.transformMatrix():";
-          view_params.transformMatrix().dump();
-          FML_LOG(FATAL) << "view_mutators.total_transform "
-                            "!= view_params.transformMatrix()";
+        // Use built-in get method for SkMatrix to get values
```

Comment:
A comment first like `// Verify that we're unpacking the mutators' transform matrix correctly on debug builds.` might be useful.